packages/nimble-components/src/table/components/header/template.ts

```diff
@@ -15,5 +16,8 @@ export const template = html<TableHeader>`
         ${when(x => x.sortDirection === TableColumnSortDirection.descending, html`
             <${iconArrowDownTag} class="sort-indicator" aria-hidden="true"></${iconArrowDownTag}>
         `)}
+        ${when(x => x.isGrouped, html`
```

Comment:
I don't think I saw any tests that verify that this icon is shown/hidden at the appropriate times.

Reply:
Added.